src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -249,27 +247,14 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
 
   const totalPages = callPages.length;
 
-  const isSpaceOrEnterKey = (event: React.KeyboardEvent<HTMLButtonElement>) =>
-    [KEY.ENTER, KEY.SPACE].includes(event.key);
-
-  const handleToggleCameraKeydown = (event: React.KeyboardEvent<HTMLButtonElement>) => {
-    if (isSpaceOrEnterKey(event)) {
-      toggleCamera(call);
-    }
-
-    return true;
-  };
-
```

Comment:
This logic can be achieved with the `handleKeyDown()` keyboard util
`handleKeyDown()` execute the callback in arguments on a key press of spacebar or enter